src/styles/main.scss

```diff
@@ -1,3 +1,56 @@
-body {
-  margin: 0;
+@import "./utils/variables.scss";
+@import "./utils/mixins.scss";
+@import "./reset.scss";
+
+html {
+  font-family: Arial, sans-serif;
+}
+
+.page {
+  display: flex;
+  justify-content: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: calc((#{$block-size} + 1px) * 7 + (#{$calendar-padding} * 2));
+  gap: 1px;
+  padding: $calendar-padding;
+  padding-top: 50px;
+
+  @for $i from 28 to 31 {
+    &--month-length--#{$i} >:nth-child(n + #{$i + 1}) {
+      display: none;
+    }
+  }
+
+  @each $name, $index in $days {
+    &--start-day--#{$name} >:first-child {
+      margin-left: calc((#{$block-size} + 3px) * #{$index});
```

Comment:
you don't have to use this syntax. you can just `($block-size + 3px) * $index`

Reply:
and better move 3px to a variable

Reply:
same for all scss calculations

Reply:
It doesn't work like that:(